scripts/qa_crowbarsetup.sh

```diff
@@ -2450,6 +2450,12 @@ function enable_ssl_generic()
             fi
             $p "$a['api']['protocol']" "'https'"
         ;;
+        manila)
+            if ! iscloudver 7plus ; then
```

Comment:
thanks @jdsn Im gonna wait for https://github.com/SUSE-Cloud/automation/pull/1078 so a consensus is reached before changing this again.